src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -442,6 +442,7 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_staticsite_environments')
         g.custom_show_command('show', 'show_staticsite_environment')
         g.custom_command('functions', 'list_staticsite_functions')
+        g.custom_command('delete', 'delete_staticsite_environment')
```

Comment:
Do we need to consider adding the `confirmation=True` for this command?

Reply:
Good call, I've added it!